### PR TITLE
Flip version in doc [branch-0.10]

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

**Review by glorysdj — APPROVED** [review]
changes in dockerfiles LGTM

File: docs/docs/APIGuide/TFPark/gan-estimator.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ estimator.train(input_fn=input_fn, end_trigger=MaxIteration(100))
 
 * **input_fn**: a python function that takes zero arguments and return a TFDataset. Each record in the TFDataset should
 a tuple. The first element of the tuple is generator inputs, and the second element of the tuple should be real data.
-* **end_trigger**: BigDL's [Trigger](https://bigdl-project.github.io/0.9.0/#APIGuide/Triggers/) to indicate when to stop the training. If none, defaults to
+* **end_trigger**: BigDL's [Trigger](https://bigdl-project.github.io/0.10.0/#APIGuide/Triggers/) to indicate when to stop the training. If none, defaults to
```
Comment (hkvision) [inline docs/docs/APIGuide/TFPark/gan-estimator.md:50]: bigdl should be 0.12.2?

File: docs/docs/APIGuide/TFPark/tf-optimizer.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ set_val_summary(summary)
 * **summary**: The validation summary to be set. A ValidationSummary object contains information
                necessary for the optimizer to know how often the logs are recorded,
                where to store the logs and how to retrieve them, etc. For details,
-               refer to the docs of [ValidationSummary](https://bigdl-project.github.io/0.9.0/#ProgrammingGuide/visualization/).
+               refer to the docs of [ValidationSummary](https://bigdl-project.github.io/master/#ProgrammingGuide/visualization/).
```
Comment (hkvision) [inline docs/docs/APIGuide/TFPark/tf-optimizer.md:119]: Why here is master but above is a specific version? Better keep them the same.

File: docs/docs/APIGuide/TFPark/gan-estimator.md (hunk 1 above)
Comment (Le-Zheng) [reply docs/docs/APIGuide/TFPark/gan-estimator.md]: Upated.

**Pull request merged**